plugins/TouchIcon/class.touchicon.plugin.php

```diff
@@ -45,7 +45,7 @@ public function OnDisable() {
    public function Structure() {
       // Backwards compatibility with v1.
       if (C('Plugins.TouchIcon.Uploaded')) {
-         SaveToConfig('Garden.TouchIcon', PATH_ROOT.'/uploads/TouchIcon/apple-touch-icon.png');
+         SaveToConfig('Garden.TouchIcon', 'TouchIcon/apple-touch-icon.png');
```

Comment:
This is wrong. It should be blank for the default path or else you will be calling `Gdn_Upload::Url()` on a path that isn't in the uploads folder.